hadoop-ozone/tools/src/main/java/org/apache/hadoop/ozone/freon/containergenerator/GeneratorDatanode.java

```diff
@@ -256,7 +256,7 @@ public void generateData(long index) throws Exception {
         //loop to create multiple chunks per blocks
         while (writtenBytes < getKeySize()) {
           int currentChunkSize =
-              Math.min(getKeySize() - writtenBytes, chunkSize);
+                  Math.min(getKeySize() - writtenBytes, chunkSize);
```

Comment:
`GeneratorDatanode` only has whitespace change.  Can you please omit it from the patch?

Reply:
Yea sorry about that, I have removed it